Independ performance test

TingDaoK opened the pull request:
*Issue #, if available:*

*Description of changes:*
- Use the S3 asset as the bucket to test performance, and upload a 5GB object before download test. Now, we can run benchmark stuff in any account!

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

rccarper reviewed (commented):
What happened to the benchmark-config.json file? :)  Otherwise, everything looks fine.

TingDaoK commented:
> What happened to the benchmark-config.json file? :) Otherwise, everything looks fine.

https://github.com/awslabs/aws-c-s3/pull/133/files#diff-32260354bf9172bf43dc447a4c3d31af84703c9b11e6bd7c68ecb9df5c81beaf? Do you mean I deleted this file? It's because I only want one configuration file in benchmark, added gitignore [here](https://github.com/awslabs/aws-c-s3/pull/127/files#diff-c4b6ce0388aa8911a96845486d77973747e65e8e9e5f4c0d60e0848d604d9db7R11-R12). It will be copied from [here](https://github.com/awslabs/aws-c-s3/pull/127/files#diff-ea7af9eb7575da18deee76c583ecc4f0a4cbccbcd5ac7df83153ba067a1ed667R204-R213)
